e2e/test/scenarios/dashboard/chained-filters.cy.spec.js

```diff
@@ -157,12 +158,9 @@ describe("scenarios > dashboard > chained filter", () => {
         });
 
         // Add previously created question to the dashboard
-        cy.request("POST", `/api/dashboard/${DASHBOARD_ID}/cards`, {
-          cardId: QUESTION_ID,
-          row: 0,
-          col: 0,
-          size_x: 8,
-          size_y: 6,
```

Comment:
Yup, so in most places as I changed from manually sending the request to using the helper I tried to remove the explicit `size_x` and `size_y` definition, since usually it wasn't necessary (the tests worked fine with an 8x8 card). There were a few cases where it did need a specific size so I left those unchanged.